CHANGELOG.md

```diff
@@ -1,3 +1,55 @@
+# v1.4.0 (2021-11-12)
+
+## OS Changes
+
+* Add 'apiclient exec' for running commands in host containers ([#1802], [#1790])
+* Improve boot performance by 5-8 seconds ([#1809])
```

Comment:
Most of the improvement is specific to the `aws-k8s` variants, which I should've been more specific about in the PR. `aws-ecs-1` didn't take the hit from copying CNI plugins or having kubelet wait for the docker daemon, and won't see as much benefit.

```suggestion
* Improve boot performance for recent Kubernetes variants([#1809])
```